karma.config.d/karma.conf.js

```diff
@@ -4,6 +4,10 @@ module.exports = function(config) {
         // default timeout 60_000, but looks like it is not enough for macOS on GitHub runner
         // increasing the timeout should help to reduce number of failures
         captureTimeout: 120_000,
+        // the default value is 20_000
```

Comment:
The error message mentions 60 s so I'm afraid it won't help, but let's try anyway!